src/main/scala/codacy/codenarc/docs/ResourceHelper.scala

```diff
@@ -0,0 +1,65 @@
+package codacy.codenarc.docs
+
+import java.io.InputStream
+import java.net.URL
+import java.nio.charset.{CodingErrorAction, StandardCharsets}
+import java.nio.file.{Files, Path, StandardOpenOption}
+
+import scala.io.{Codec, Source}
+import scala.util.{Failure, Properties, Try}
+
+object ResourceHelper {
+
+  implicit val codec: Codec = Codec.UTF8
```

Comment:
Use brackets to scope the configs of the codec to it's definition

Reply:
Done